server/db/schema/students.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE Student (
+    id INTEGER NOT NULL PRIMARY KEY,
+    level ENUM('beginner', 'intermediate', 'advanced') NOT NULL,
+    CONSTRAINT fk_user_student FOREIGN KEY (id) REFERENCES User (id)
```

Comment:
In general whenever we use foreign keys we should be cascading on delete, can you add that?